client/src/components/CreateJob/createjob.js

```diff
@@ -48,51 +52,91 @@ class CreateJob extends Component {
-        // .then(res => this.loadJobs())
+        .then(res => this.loadJobs())
         .catch(err => console.log(err));
     }
   };
 
   render() {
-    return (
-      <Container fluid>
-        <Row>
-          <Col size="md-6">
-            <Jumbotron>
-              <h1>Create Job</h1>
-            </Jumbotron>
-            <form>
-              <Input
-                type="text"
-                value={this.state.title}
-                onChange={this.handleInputChange}
-                name="title"
-                placeholder="Title (required)"
-              />
-              <TextArea
-                value={this.state.description}
-                onChange={this.handleInputChange}
-                name="description"
-                placeholder="Description (required)"
-              />
-              <Input
-                type="number"
-                value={this.state.budget}
-                onChange={this.handleInputChange}
-                name="budget"
-                placeholder="Budget (Optional)"
-              />
-              <FormBtn
-                disabled={!(this.state.description && this.state.title)}
-                onClick={this.handleFormSubmit}
-              >
-                Submit Job
-              </FormBtn>
-            </form>
-          </Col>
-        </Row>
-      </Container>
-    );
+    if (!this.props.loggedIn) {
+      return <Redirect to="/login" />;
+    } else {
+      return (
+        <div className="container">
+          <Container fluid>
+            <Row>
+              <Col size="md-6">
+                <Jumbotron>
+                  <h1>Create Job</h1>
+                </Jumbotron>
+                <form>
+                  <Input
+                    type="text"
+                    value={this.state.title}
+                    onChange={this.handleInputChange}
+                    name="title"
+                    placeholder="Title (required)"
+                  />
+                  <TextArea
+                    value={this.state.description}
+                    onChange={this.handleInputChange}
+                    name="description"
+                    placeholder="Description (required)"
+                  />
+                  <Input
+                    type="number"
+                    value={this.state.budget}
+                    onChange={this.handleInputChange}
+                    name="budget"
+                    placeholder="Budget (Optional)"
+                  />
+                  <FormBtn
+                    disabled={!(this.state.description && this.state.title)}
+                    onClick={this.handleFormSubmit}
+                  >
+                    Submit Job
+                  </FormBtn>
+                </form>
+              </Col>
+              <Col size="md-6 sm-12">
+                <Jumbotron>
+                  <h1>Job List</h1>
+                </Jumbotron>
+                {this.state.jobs.length ? (
+                  <List>
```

Comment:
This list can be extracted out to another component. To me, it looks like you guys have like 3 to 4 components in this 1 component. This guy that can be broken up a lot more